apps/frontend/components/FormValidation/FloodFormValidation/FloodFormValidation.tsx

```diff
@@ -62,7 +75,21 @@ export const FloodFormValidation = ({
     }
   }, [shouldReset, reset]);
 
-  const onSubmit = (data: unknown) => console.log(data);
+  const onSubmit = (data: FloodFormType) => {
+    const triggerAndUpdateDefault = async () => {
+      try {
+        const status = await trigger(formatFloodFormToRaw(data));
+        if (status === 201) {
+          reset(data);
```

Comment:
should we reset the data? it would be a really weird experience for the user to submit their edits and then have it reset back to what it used to look like

Reply:
It is reset with Data so we update the defaults